src/main/resources/app.conf

```diff
@@ -347,34 +381,45 @@ app {
         # the initial delay before retrying that will be multiplied with the 'factor' for each attempt
         initial-delay = ${app.key-value-store.retry.initial-delay}
         initial-delay = ${?REALMS_KEY_VALUE_STORE_RETRY_INITIAL_DELAY}
+        # the maximum delay applied to the retrying strategy
+        max-delay = ${app.sourcing.retry.max-delay}
         # maximum number of retries in case of failure (applicable only for strategy "exponential")
         max-retries = ${app.key-value-store.retry.max-retries}
         max-retries = ${?REALMS_KEY_VALUE_STORE_RETRY_MAX_RETRY}
         # the exponential factor (applicable only for strategy "exponential")
         factor = ${app.key-value-store.retry.factor}
         factor = ${?REALMS_KEY_VALUE_STORE_RETRY_FACTOR}
+        # the linear increment (applicable only for strategy "linear")
+        increment = ${app.sourcing.retry.increment}
       }
     }
 
     # realms indexing configuration
     indexing {
       # maximum number of events taken on each batch
-      batch = ${app.indexing.batch}
-      batch = ${?REALMS_INDEXING_BATCH}
+      batch-chunk = ${app.indexing.batch-chunk}
```

Comment:
batch and chunk mean similar things; either keep the name `batch` or rename to `batch-size`